sessions/biases-in-delay-distributions.qmd

```diff
@@ -217,12 +224,13 @@ Can you establish under which conditions the bias in estimation gets worse?
 # Real-time estimation: truncation
 
 The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
-However, information on delay distribution is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
+However, information on delay distributions is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
+
+Estimating delays in real time comes with particular challenges, as we know that our data are always incomplete: not enough time has passed to observe all of the relevant events. We call this incompleteness *truncation*. This is a problem if we try to use incomplete secondary event data (such as recent symptom onsets) to infer the pattern of primary events (such as infections) - our estimates will be biased because we are missing some of the data.
 
-Estimating delays in real time comes with particular challenges, as the timing of the cut-off might introduce a bias.
-If, for example, infections are exponentially increasing then there will be disproportionately more people with recent symptom onset.
-Without adjustment, this would artificially decrease the estimate of the mean delay compared to its true value for all infections.
-This happens because most infections are recent (due to the exponential increase), but later symptom onsets amongst these have not had a chance to happen yet.
+This bias also changes over the course of an outbreak. If, for example, infections are exponentially increasing, then there will be disproportionately more people with recent symptom onset.
+Without adjustment, this would artificially decrease the estimate of the mean delay compared to its true value for all infections that have in fact already occurred.
+This happens because most infections are recent (due to the exponential increase), but later symptom onsets amongst these have not had a chance to happen yet. So among a cohort of people infected at the same time, those with shorter delays to onset would be over-represented.
```

Comment:
```suggestion
This happens because most infections are recent (due to the exponential increase), but later symptom onsets amongst these have not had a chance to happen yet.
So among a cohort of people with symptom onset at the same time, those with shorter delays to onset would be over-represented.
```
(the bias occurs when looking backwards)

Reply:
oops, thanks!